Cargo.toml

```diff
@@ -242,13 +242,18 @@ orml-traits = { git = "https://github.com/open-web3-stack/open-runtime-module-li
 orml-xcm-support = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v1.1.0", default-features = false }
 orml-xtokens = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v1.1.0", default-features = false }
 
+# Other pallets
+# (wasm)
+substrate-fixed = { git = "https://github.com/encointer/substrate-fixed", default-features = false }
```

Comment:
what's this dependency for ?

Reply:
crate for fixed point arithmetic used by the `parachain-staking` pallet

Reply:
it's published on crates.io : https://crates.io/crates/substrate-fixed

Reply:
https://github.com/encointer/substrate-fixed/pull/20